apps/web/src/types/pool.ts

```diff
@@ -1,24 +1,7 @@
 import { IconType } from "@gc-digital-talent/ui";
-import {
-  Pool,
-  Classification,
-  Maybe,
-  UserPublicProfile,
-} from "@gc-digital-talent/graphql";
 
 import { Status, StatusColor } from "~/components/StatusItem/StatusItem";
 
-export type SimpleClassification = Pick<Classification, "group" | "level">;
```

Comment:
I was trying to fix an issue with type inference on one of the forms and while doing that I removed these to see if they were causing problems. They weren't but at the same time I realized they weren't really doing anything since everything on `Pool` and `Classification` is optional.

I can roll it back in if we find value or think it expands the scope too much though :+1:

Reply:
It is simpler, for sure.  I think we're going to need these reduced types less with the query fragments in the new codegen preset, too.  Let's keep this change.